### PR TITLE
Add to the docs in the README - more about how things work, how to import licenses

### PR DESCRIPTION
**Pull request by dpoirier**
## Description
Add to the docs in the README - more about how things work, how to import licenses

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -92,26 +83,148 @@ Or, on a custom port and address::
...
+-----------------
+
+This project is not intended to serve the license and deed pages directly.
+Though if it's deployed on a public server it could do that, performance
+would probably not be acceptable.
+
+Instead, a command line tool can be used to save all the rendered HTML
+pages for licenses and deeds as files. Then those files are used as part
+of the real creativecommons.org site, just served as static files.
+See details farther down.
```
Comment (dpoirier) [inline README.rst:96]: still to be written...

Comment (dpoirier) [reply README.rst:96]: I've now added the directions for saving the static files from the site.

File: README.rst [hunk 2]
```diff
@@ -92,26 +83,149 @@ Or, on a custom port and address::
...
+the 3.0 version of the BY license terms as ported to the Armenia jurisdiction.
+
+A License can exist in multiple languages or translations. Each one, including
+English, is represented by a LegalCode record. A LegalCode is identified by
+a license and a language, e.g. we might refer to the "BY 3.0 Armenia in Armenian"
+legalcode record.
+
+For some licenses, the text is in gettext files (.po and .mo) in the cc-licenses-data
+repository. For others, the text is stored in the database, directly in
+LegalCode records.
```
Comment (Audiosutras) [inline README.rst:123]: I think we should mention which license's text is stored in cc_licenses_data and which is stored in the database.

File: README.rst [hunk 3]
```diff
@@ -92,26 +83,149 @@ Or, on a custom port and address::
...
     python manage.py load_html_files ../creativecommons.org/docroot/legalcode
 
 It will read the HTML files from the specified directory, populate the database
-with LegalCode and License records, and at least for the BY 4.0 licenses, create
-.po and .mo files under locale.licenses.
+with LegalCode and License records, and create
+.po and .mo files in cc-licenses-data.
 
-To clean things up ready to start over, you can run clear_license_data again.
+Now commit the changes from cc-licenses-data and push to Github.
```
Comment (Audiosutras) [inline README.rst:225]: Should we reference `publish` here?

**Review by Audiosutras — APPROVED** [review]
Really Nice docs. Very well thought out! 📖

**Pull request merged**